Vizzuality/heco-invest
# feat(front): dashboard users table [LET-631]

This PR contains:

A table of users to show on dashboard to let user as a project developer owner invite new users.
A Confirmation Prompt component apart from HeCo styles.

## Testing instructions

API  integration to see real data users and be able to delete users is not in this PR. We are using mock data. I will open a new task for integration issues:

![Screenshot 2022-06-21 at 17 38 29](https://user-images.githubusercontent.com/51995866/174840670-9f5de984-f193-4dcc-9deb-a54350f3aa9b.png)

## Tracking
[LET-631](https://vizzuality.atlassian.net/browse/)
